src/main/java/org/xstefank/model/yaml/Format.java

```diff
@@ -4,7 +4,7 @@
 
 public class Format {
 
-    private String title;
+    private String title, commit;
```

Comment:
Again, I need to setup checkstyle, but please use one line for definition of attributes